api/wan/read

```diff
@@ -77,7 +77,20 @@ if($cmd eq 'providers') {
     foreach (qw(MaxNumberPacketLoss MaxPercentPacketLoss NotifyWan PingInterval WanMode)) {
         $ret->{'configuration'}{'multiwan'}{$_} = $cdb->get_prop('firewall',$_);
     }
-    $ret->{'configuration'}{'multiwan'}{'EmailAddress'} = $cdb->get_prop('root', 'EmailAddress');
+    my $NotifyWanTo =  $cdb->get_prop('firewall','NotifyWanTo');
+    $NotifyWanTo  =~ s/,/ /g;
+    my $EmailNotifications = $cdb->get_prop('root', 'EmailAddress');
+    $EmailNotifications =~ s/,/ /g;
+
+    # default property is NotifyWanTo === root@localhost
+    if ($NotifyWanTo ne "" && $NotifyWanTo ne 'root@localhost') {
```

Comment:
I think the condition `$NotifyWanTo ne 'root@localhost'` should never be met because `root@localhost` is never saved inside `NotifyWanTo` prop: am I wrong?

Reply:
`root@localhost` is the default value of NotifyWanTo and NotifyWanFrom, so we met them at the first installation

https://github.com/NethServer/nethserver-firewall-base/blob/master/root/etc/e-smith/db/configuration/defaults/firewall/NotifyWanTo

Reply:
`root@localhost` is the current default from db defaults.

Thank you @stephdl  for pointing it out